dns/py/params.py

```diff
@@ -276,6 +276,15 @@ def iterate(remaining, pv):
 
     return cases
 
+  def add_param(self, param_name, values):
```

Comment:
semantics for this seem weird -- it won't add unless it already exists?

Reply:
good point, renamed it to set_param